src/nativeAssetManager.js

```diff
@@ -244,8 +244,9 @@ export function newNativeAssetManager(win) {
     let html = document;
 
     (assets || []).forEach(asset => {
-      const searchString = (adId) ? `${NATIVE_KEYS[asset.key]}:${adId}` : `${NATIVE_KEYS[asset.key]}`
-      html = html.replace(searchString, asset.value);
+      const searchString = (adId) ? `${NATIVE_KEYS[asset.key]}:${adId}` : `${NATIVE_KEYS[asset.key]}`;
+      const searchStringRegex = new RegExp(searchString, "g");
```

Comment:
Could you change the double quotes surrounding `g` to single quotes, the reset of the change is good.